godtools/TractClasses/ManifestProperties.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ManifestProperties.swift
+//  godtools
+//
+//  Created by Pablo Marti on 6/9/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import UIKit
+
+class ManifestProperties: XMLNode {
```

Comment:
I see this class is used as an optional throughout many other classes. It seems we should be guaranteed an instance of this class, even if all the properties it contains are not present. It would be better and cleaner, in my opinion if this weren't passed around as an optional and force unwrapped.